calabash-cucumber/lib/calabash-cucumber/wait_helpers.rb

```diff
@@ -231,6 +231,27 @@ def wait_for_elements_exist(elements_arr, options={})
         end
       end
 
+      # Waits for any of one or more Calabash queries to return a non-empty result
+      # Uses `wait_for`.
+      # @see #wait_for
+      # @see #wait_for_any_element_exists
+      # @see Calabash::Cucumber::WaitHelpers::DEFAULT_OPTS
+      #
+      # @param [Array<String>] elements_arr an Array of Calabash queries to wait for (i.e. `element_exists(element_query)`)
+      # @param [Hash] options options for controlling the details of the wait.
+      #   The same options as {Calabash::Cucumber::WaitHelpers::DEFAULT_OPTS} apply.
+      # @return [nil] when the condition is satisfied
+      # @raise [Calabash::Cucumber::WaitHelpers::WaitError] when the timeout is exceeded
+      def wait_for_any_elements_exist(elements_arr, options={})
+        if elements_arr.is_a?(String)
+          elements_arr = [elements_arr]
```

Comment:
from my point of view that part not needed we can simple expect always a Array since method called `wait_for_any_elements_exist`
if you need to wait for one element please use `wait_for_element_exists`